x/ecocredit/server/core/msg_create_class_test.go

```diff
@@ -212,41 +220,48 @@ func (s *createClassSuite) ExpectTheResponse(a gocuke.DocString) {
 }
 
 func (s *createClassSuite) createClassExpectCalls() {
-	var allowlistEnabled bool
-	var allowedClassCreators []string
-	var creditClassFee sdk.Coins
-
-	s.paramsKeeper.EXPECT().
-		Get(s.sdkCtx, core.KeyAllowlistEnabled, &allowlistEnabled).
-		Do(func(ctx sdk.Context, key []byte, allowlistEnabled *bool) {
-			*allowlistEnabled = s.params.AllowlistEnabled
-		}).
-		AnyTimes() // not expected on failed attempt
+	err := s.stateStore.AllowListEnabledTable().Save(s.ctx, &api.AllowListEnabled{
+		Enabled: s.allowlistEnabled,
+	})
+	require.NoError(s.t, err)
 
-	s.paramsKeeper.EXPECT().
-		Get(s.sdkCtx, core.KeyAllowedClassCreators, &allowedClassCreators).
-		Do(func(ctx sdk.Context, key []byte, allowedClassCreators *[]string) {
-			*allowedClassCreators = s.params.AllowedClassCreators
-		}).
-		AnyTimes() // not expected on failed attempt
+	for _, creator := range s.allowedClassCreators {
+		addr, err := sdk.AccAddressFromBech32(creator)
+		require.NoError(s.t, err)
 
-	s.paramsKeeper.EXPECT().
-		Get(s.sdkCtx, core.KeyCreditClassFee, &creditClassFee).
-		Do(func(ctx sdk.Context, key []byte, creditClassFee *sdk.Coins) {
-			*creditClassFee = s.params.CreditClassFee
-		}).
-		AnyTimes() // not expected on failed attempt
+		err = s.stateStore.AllowedClassCreatorTable().Save(s.ctx, &api.AllowedClassCreator{
+			Address: addr,
+		})
+		require.NoError(s.t, err)
+	}
```

Comment:
Same here. This is not a mock expect call. This should be included in `AliceIsAnApprovedCreditClassCreator`.